packages/archetype/src/__demo__/examples/Default.js

```diff
@@ -15,9 +15,10 @@
  */
 
 /* @flow */
-import React from 'react';
 import Archetype from '../../Archetype';
 
-export default function Default() {
-  return <Archetype />;
-}
+export default {
```

Comment:
Since this file (along with all other `/src/__demo__/examples/*.js` files) is no longer a component, maybe the filename should be lower, kebab-case?